scalajslib/src/mill/scalajslib/ScalaJSWorkerApi.scala

```diff
@@ -0,0 +1,72 @@
+package mill.scalajslib
+
+import java.io.File
+
+import mill.api.{Ctx, Result}
+import mill.define.Discover
+import mill.scalajslib.api._
+import mill.{Agg, T}
+
+@deprecated("Use mill.scalajslib.worker.ScalaJSWorker instead", since = "mill 0.10.4")
+class ScalaJSWorker extends AutoCloseable {
```

Comment:
Deprecated and implemented on behalf of the other `@internal private[scalajslib]` worker I created.
We can delete it on Mill 0.11